inc/reminder.class.php

```diff
@@ -878,6 +883,22 @@ static function showListForCentral($personal = true) {
          }
       }
 
+      if (ReminderTranslation::isReminderTranslationActive()
+          && countElementsInTable('glpi_remindertranslations') > 0) {
```

Comment:
This `countElementsInTable('glpi_remindertranslations') > 0` will add an aditionnal query, and does not seems really usefull.

Reply:
Corrected thx